routes/index.js

```diff
@@ -311,11 +302,15 @@ var routes = {
       location = "thunderbird/" + location;
     }
     var frequency = request.params.frequency || 'single';
+    var options = {
+      recurring: frequency === 'monthly',
+      accountType: request.params.accountType
+    };
     var request_id = request.headers['x-request-id'];
     if (frequency !== 'monthly') {
-      paypal.getSingleCheckoutDetails({
+      paypal.getCheckoutDetails({
         token: request.url.query.token
-      }, function(err, checkoutDetails) {
+      }, options, function(err, checkoutDetails) {
```

Comment:
Anywhere where I add `options` I'm adding functionality to send messages about the transaction. I believe I do it in 4 places.